src/HomePage.jsx

```diff
@@ -1,15 +1,50 @@
 import { Link } from 'react-router-dom';
 
+import styled from '@emotion/styled';
+
+const Title = styled.h2({
+  margin: 0,
+  fontSize: '2em',
+  padding: '.4em 0',
+});
+
+const List = styled.ul({
+  margin: 0,
+  padding: 0,
+  listStyle: 'none',
+  display: 'flex',
+});
+
+const Item = styled.li({
+  marginRight: '1em',
+  '& a': {
+    color: '#333',
+    textDecoration: 'none',
+  },
+  '& a:hover': {
+    fontWeight: 'bold',
+    color: '#000',
+  },
+});
+
 export default function HomePage() {
+  const lists = [
+    ['/about', 'About'],
+    ['/login', 'Login'],
+    ['/restaurants', 'Restaurants'],
+    ['/xxx', '멸망의 길'],
+  ];
+
   return (
     <div>
-      <h2>Home</h2>
-      <ul>
-        <li><Link to="/about">About</Link></li>
-        <li><Link to="/login">Log in</Link></li>
-        <li><Link to="/restaurants">Restaurants</Link></li>
-        <li><Link to="/xxx">멸망의 길</Link></li>
-      </ul>
+      <Title>Home</Title>
+      <List>
+        {lists.map((list) => (
+          <Item>
+            <Link to={list[0]}>{list[1]}</Link>
+          </Item>
+        ))}
```

Comment:
list[0], list[1]이 무엇인지 이해하기 매우 어렵습니다.

```suggestion
        {lists.map(([link, text]) => (
          <Item>
            <Link to={link}>{title}</Link>
          </Item>
        ))}
```